### DISCUSSION TITLE
Can I print the Chinese str in the online terminal?

### DISCUSSION THREAD

**Discussion by angzicn** [pybricks/support#1742] [Q&A] [answered]
Thank you, I'm a Chinese pybricks fan and I want to teach my son use pybricks to make python code.
Can i print the Chinese str in the online terminal?
There is something wrong with the Chinese str printing in the terminal.
incomplete display,  like this↓
![微信截图_20240806152205](https://github.com/user-attachments/assets/7a03c56e-28d3-4696-a319-fdec58153c93)
thank you！

**Comment by BertLindeman** [comment]
Hello,

Printing special characters to the terminal is not so straightforward.
I tried printing an arrow character and if I did a short wait after printing that character it showed OK.

After printing an utf8 character I added a 10 milli second wait:
```python
wait(10)
```
and that helped me.

Bert

**Comment by dlech** [comment, marked as answer]
Thanks for reporting this. I filed a bug report so that we can fix this.

https://github.com/pybricks/support/issues/1743

**Reply by angzicn** [reply]
thanks a lot for all your help
